hotrod/pom.xml

```diff
@@ -240,8 +240,8 @@
                         <phase>package</phase>
                         <configuration>
                             <target>
-                                <!-- We will just take the JDK11-specific classes and move them to regular locations
-                                     as Hyperfoil is JDK11+ only anyway. We won't need to mess with multi-release JAR then -->
+                                <!-- We will just take the JDK17-specific classes and move them to regular locations
+                                     as Hyperfoil is JDK17+ only anyway. We won't need to mess with multi-release JAR then -->
                                 <unzip src="${project.build.directory}/hyperfoil-hotrod-shaded.jar" dest="${project.build.directory}/shaded" />
                                 <move todir="${project.build.directory}/shaded/shaded">
                                     <fileset dir="${project.build.directory}/shaded/META-INF/versions/11/" />
```

Comment:
what's going on here? I see that things are shared into `11`?

Reply:
Oh whoops meant to put 17. TBH, this code might be able to be deleted as I don't see a versions/11 or versions/17 directory after a build is ran.

Reply:
Hey @franz1981 @wburns  IIUC we should remove the `move` entry here because with this pull request, Hyperfoil will become JDK17+ only. Thus, moving everything from `${project.build.directory}/shaded/META-INF/versions/11/` to `${project.build.directory}/shaded/shaded` will override `jdk17` classes with `jdk11`-specific ones which is not what we want. wdyt?

Reply:
It isn't an issue and I actually completely remove it in https://github.com/Hyperfoil/Hyperfoil/pull/349